src/mkindex.hpp

```diff
@@ -202,10 +192,7 @@ void realMain(LambdaIndexerOptions     const & options)
     // see if final sequence set actually fits into index
 //     checkIndexSize(translatedSeqs, options, seqan::BlastProgramSelector<p>());
 
-    std::tie(f.redSeqs, f.index) =
-        generateIndex<c_dbIndexType == DbIndexType::BI_FM_INDEX, c_redAlph>(f.transSeqs, options);
-
-
+    f.index = generateIndex<c_dbIndexType == DbIndexType::BI_FM_INDEX>(f.transSeqs, options);
```

Comment:
This means we always have a bidirectional index?

Reply:
no, it's a `==` not a `=`

Reply:
Oh right